### DISCUSSION TITLE
How to handle oAuth server validation errors

### DISCUSSION THREAD

**Discussion by jahidanowar** [nuxt-community/auth-module#1449] [Q&A]
```js
async loginWithFb() {
      try {
        const response = await this.$auth.loginWith('facebook')
        console.log(response)
      } catch (err) {
        console.log(err.response.data)
      }
    },
```

I have returned a 400 error against the `token` endpoints call but the above code is not handling the error.

What's the proper way of handling an error like this.  

Basically, I am trying to restrict the login to only those who have an email attached to their FB. I did this in the backend but I am not able to print the message to the frontend.

Auth Config
```js
auth: {
    strategies: {
      facebook: {
        clientId: 'XXXXXXXXX',
        responseType: 'code',
        scope: ['public_profile', 'email'],
        codeChallengeMethod: '',
        endpoints: {
          token: `${process.env.BACKEND}/user/social-login/facebook`,
          userInfo: `${process.env.BACKEND}/user/me`,
        },
      },
    },
    rewriteRedirects: true,
    redirect: {
      logout: '/',
      home: false,
    },
  },
```

**Comment by steklopod** [comment]
It's not about a rest request-response. It's about redirects. So I think there is no way to handle it on the frontend  (but I am not 100% sure about it)
